pages/coalition.html

```diff
@@ -0,0 +1,202 @@
+			</br>
+				It’s like the Common App for social startups: one application, over 30 participating companies.
+			</p>
+		</div>
+		<div class="ill col-md-3 col-12 mt-5 mb-5 mb-md-0 text-center">
+			<img alt="Find an Internship" width="300px" src="/assets/images/illustrations/find_an_internship.svg"/>
+		</div>
+		</div>
+	</div>
+</section>
+
+<section class="py-0" style="margin-top:-100px" >
+	<img alt="Bolt" class="accent d-none d-md-inline" style="width: 100px; position:relative; bottom:50px; left: 100px;" src="/assets/images/accents/bolt.svg" />
+	<img alt="Blob" class="blob" style="width: 2200px; left:-500px; position:absolute; z-index: -1;" src="/assets/images/accents/blob.svg" />
+
+	<div class="container">
+		<div class="row py-5">
+		<div class="col-md-4 col-12"></div>
+		<div class="col-md-8 col-12">
+			<h2>The 3 Step Process</h2>
+		</div>
+		</div>
+		<div class="row mb-5">
+		<div class="col-md-3 col-12 mb-4 mb-md-0">
+			<img alt="Step 1" width="190px" class="ill" src="/assets/images/icons/submit_an_app.svg" />
+		</div>
+		<div class="col-md-1 col-3 text-center">
+			<img alt="Step 1" width="50px" src="/assets/images/accents/coalition-1.svg" />
+		</div>
+		<div class="col-md-8 col-9">
+			<h3 style="color: black">Submit an application</h3>
+			<p>There are no official eligibility requirements, but the program is designed for those with strong software engineering experience.</p>
+		</div>
+		</div>
+		<div class="row mb-5">
+		<div class="col-md-3 col-12 mb-4 mb-md-0">
+			<img alt="Step 2" width="220px" class="ill" src="/assets/images/icons/impact_labs_interview.svg" />
+		</div>
+		<div class="col-md-1 col-3 text-center">
+			<img alt="Step 2" width="50px" src="/assets/images/accents/coalition-2.svg" />
+		</div>
+		<div class="col-md-8 col-9">
+			<h3 style="color: black">Impact Labs interview</h3>
+			<p>If selected, you'll have the opportunity to interview with the Impact Labs team to determine your fit within the participating companies in the coalition.</p>
+		</div>
+		</div>
+		<div class="row mb-5">
+		<div class="col-md-3 col-12 mb-4 mb-md-0">
+			<img alt="Step 3" width="220px" class="ill" src="/assets/images/icons/company_interview_1.svg" />
+		</div>
+		<div class="col-md-1 col-3 text-center">
+			<img alt="Step 3" width="50px" src="/assets/images/accents/coalition-3.svg" />
+		</div>
+		<div class="col-md-8 col-9">
+			<h3 style="color: black">Individual company interview</h3>
+			<p>The last step: you'll select your top 5 preferences of the companies within the coalition and interview with them. Salary, start date, and role are all finalized at this time.</p>
+		</div>
+		</div>
+	</div>
+</section>
+
+<section>
+	<img alt="Plus" class="accent d-none d-md-inline" style="height: 130px; width: 130px; position:relative; float:right; bottom:100px; right: 100px;" src="/assets/images/accents/plus.svg" />
```

Comment:
Could possibly inline these SVGs now that we have liquid syntax! [Found a slim plugin for this here](https://github.com/sdumetz/jekyll-inline-svg)

Reply:
Agreed, theres all sorts of Jekyll plugins we can add if we want!